packages/backend/src/modules/tvl/utils/createAmountId.ts

```diff
@@ -9,6 +9,7 @@ export function createAmountId(amountConfig: AmountConfigEntry): AmountId {
   input.push(amountConfig.chain)
   input.push(amountConfig.project.toString())
   input.push(amountConfig.type)
+  input.push(amountConfig.category)
```

Comment:
Category is not affecting the amounts fetching, should be omitted there with appropriate comment